lib/osf-components/addon/components/subscriptions/list/styles.scss

```diff
@@ -1,3 +1,25 @@
 ul {
     list-style-type: none;
+    padding: 0;
+}
+
+.SubscriptionPlaceholder {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 75%;
+    height: 45px;
+    border-top: 1px solid $color-bg-gray-dark;
+
+    &:global(.ember-content-placeholders-text) {
+        margin-top: 0 !important;
```

Comment:
I know I know, I don't like '!important' neither, but this is from an external addon and they have `!important` on that rule which prevents us from overriding it without it. There are other ways of course.

Reply:
I think we can get away with doing `border-bottom` on `.SubscriptionPlaceholder` and a `.SubscriptionPlaceholder:first-of-type { border-top: 1px solid $color-bg-gray-dark; } ` and that looks basically the same (I think it's slightly different, but it's hard to tell) as the CSS rule that's currently in place?